Minor improvements - adding video embed and switching Node --> Go (#1764)
chainguard-dev/edu: content/chainguard/chainguard-images/getting-started/nginx/index.md

In this tutorial, we will create a local demo website using nginx to serve static HTML content to a local port on your machine. Then we will use the nginx Chainguard Image to build and execute the demo in a lightweight containerized environment.

> You will need to have [nginx](https://nginx.org/en/download.html) and [Docker Engine](https://docs.docker.com/engine/install/) installed on your machine to follow along. Additionally, you can watch our [Getting Started with the nginx Chainguard Image video](https://youtu.be/KirTeDMzzxk) as you work through this tutorial.

Reviewer: ```suggestion
If you'd like, you can watch our [Getting Started with the nginx Chainguard Image video](https://youtu.be/KirTeDMzzxk) as you work through this tutorial.
```

Could you maybe add some context for the video here? Is it basically the same thing as this tutorial or are there important differences? I just wouldn't want readers to watch this video and then go through the tutorial if they're radically different.
